openlibrary/templates/type/edition/view.html

```diff
@@ -227,9 +239,11 @@ <h4 class="year">&mdash;
         </div>
         $:macros.ReadMore("subjects")
     </div>
-      $if ebooks_only:
+
+      $if editions_limit and len(editions) and len(editions) < work.edition_count:
         <p>
-          $ungettext("Showing one ebook only.", "Showing %(count)d ebooks only", len(editions), count=len(editions)) <a href="?mode=all">$_("View all %(count)d editions?", count=work.edition_count)</a>
+	  $_("Showing %(count)d featured editions", count=len(editions)).
```

Comment:
Should this be indented one more level?  Like this?
```suggestion
	    $_("Showing %(count)d featured editions", count=len(editions)).
```

Reply:
Actually, is there supposed to be an `else` block?